src/config.zig

```diff
@@ -71,12 +71,14 @@ const ConfigProcess = struct {
     clock_synchronization_window_max_ms: u64 = 20000,
     grid_iops_read_max: u64 = 16,
     grid_iops_write_max: u64 = 16,
-    grid_repair_request_max: usize = 4,
-    grid_repair_reads_max: usize = 4,
-    grid_repair_writes_max: usize = 1,
+    grid_repair_request_max: usize = 8,
+    grid_repair_reads_max: usize = 8,
+    grid_repair_writes_max: usize = 8,
     grid_cache_size_default: u64 = 1024 * 1024 * 1024,
     aof_record: bool = false,
     aof_recovery: bool = false,
+    /// When null, this defaults to message_body_size_max.
+    sync_trailer_message_body_size_max: ?usize = null,
```

Comment:
The export in `constants` (which is what other code actually uses) this is a `usize`. `?usize` here is because `message_body_size_max` is not known yet.